examples/tv-app/linux/include/application-launcher/ApplicationLauncherManager.cpp

```diff
@@ -59,42 +59,43 @@ ApplicationLaunchResponse ApplicationLauncherManager::proxyApplicationLaunchRequ
     return response;
 }
 
-static void sendResponse(const char * responseName, ApplicationLaunchResponse response)
+void sendResponse(chip::app::Command * command, ApplicationLaunchResponse response)
 {
-    emberAfFillExternalBuffer((ZCL_CLUSTER_SPECIFIC_COMMAND | ZCL_FRAME_CONTROL_SERVER_TO_CLIENT),
-                              ZCL_APPLICATION_LAUNCHER_CLUSTER_ID, ZCL_LAUNCH_APP_RESPONSE_COMMAND_ID, "us", response.status,
-                              &response.data);
-
-    EmberStatus status = emberAfSendResponse();
-    if (status != EMBER_SUCCESS)
+    CHIP_ERROR err                         = CHIP_NO_ERROR;
+    chip::app::CommandPathParams cmdParams = { emberAfCurrentEndpoint(), /* group id */ 0, ZCL_APPLICATION_LAUNCHER_CLUSTER_ID,
+                                               ZCL_LAUNCH_APP_RESPONSE_COMMAND_ID,
+                                               (chip::app::CommandPathFlags::kEndpointIdValid) };
+    chip::TLV::TLVWriter * writer          = nullptr;
+    SuccessOrExit(err = command->PrepareCommand(cmdParams));
+    VerifyOrExit((writer = command->GetCommandDataElementTLVWriter()) != nullptr, err = CHIP_ERROR_INCORRECT_STATE);
+    SuccessOrExit(err = writer->Put(chip::TLV::ContextTag(0), response.status));
+    SuccessOrExit(err = writer->PutString(chip::TLV::ContextTag(1), response.data.c_str()));
+    SuccessOrExit(err = command->FinishCommand());
+exit:
+    if (err != CHIP_NO_ERROR)
     {
-        ChipLogError(Zcl, "Failed to send %s. Error:%s", responseName, chip::ErrorStr(status));
+        ChipLogError(Zcl, "Failed to send LaunchAppResponse. Error:%s", chip::ErrorStr(err));
     }
 }
 
 EmberAfApplicationLauncherApp getApplicationFromCommand(EmberAfClusterCommand * cmd, unsigned char * reqestData,
```

Comment:
`cmd` here i unused, right?  So is `reqestData`.  Please remove those args?

Reply:
Removed.